prototype/Makefile

```diff
@@ -33,9 +33,15 @@ build_and_watch:
 client_start:
 	docker-compose -f build/deployments/docker-compose.yaml up -d client
 
+client_alpha_start:
+	docker-compose -f deployments/docker-compose.yaml up -d client_alpha
+
 ## Connect to the client daemon running in a background docker container.
 client_connect:
-	docker exec -it client /bin/bash -c "go run cmd/client/*.go"
+	docker exec -it client /bin/bash -c "go run cmd/client-with-pre-p2p/*.go"
+
+client_beta_connect:
```

Comment:
There's a client, a client alpha, a client beta, etc...

This is fine for a local working branch but I don't think it should be committed to master.

Reply:
There should be a client and a client_alpha. The client_beta is a slip and leftover from the git rebase I was doing. Thanks for the catch, will clean immediately.